app/Models/ApiKey.php

```diff
@@ -151,7 +149,7 @@ public function user(): BelongsTo
 
     public function tokenable()
     {
-        // @phpstan-ignore-next-line
+        // @phpstan-ignore return.type
         return $this->user();
     }
```

Comment:
```suggestion
    public function tokenable(): User
    {
        return $this->user();
    }
```
Why don't we specify return type ?

Reply:
This line change was automated, previously I don't know who put that in